client/src/components/Logo/Logo.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./Logo.css";
+
+// Function based React Component
+const Logo = props => {
+  // Default Class to apply to Component
+  let classList = `Logo`;
+  if (props.small) {
+    classList += ` smallLogo`;
+  }
+  if (props.clickable) {
+    return (
+      <div className={classList}>
+        <Link to="/" />
+      </div>
+    );
+  } else {
+    return <div className={classList}></div>;
+  }
```

Comment:
This is awesome! Can you make it so someone can pass in the destination URL as a prop, with the default value as `"/"`? We'll almost always want the logo to link to the homepage (if it is clickable) but there may be cases where we want to change that.

Reply:
Let's call the prop `to`, to keep it consistent with React Router